README.md

```diff
@@ -105,15 +106,15 @@ Code samples to open a connection for some typical scenarios are given below:
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.Statement;
-.
```

Comment:
Do you know why there were so many periods placed throughout the old code? Would they not break the code that's there or is there something syntax related with them that I don't know about?